### PR TITLE
Adding a display name to allow users to visualize on their choice of ID

### PR DESCRIPTION
**Pull request by dippindots**
Changing to "id (display name)" in the header and make it in the first row in various mouse over tables.
Fix https://github.com/cBioPortal/cbioportal/issues/5711.

### CODE REVIEW

File: src/pages/patientView/clinicalInformation/PatientViewPageStore.ts [hunk 1]
```diff
@@ -104,6 +104,21 @@ function transformClinicalInformationToStoreShape(patientId: string, studyId: st
         clinicalData: clinicalDataPatient
     };
     const samples = groupBySampleId(sampleIds, clinicalDataSample);
+    samples.forEach((sample, index) => {
+        let attr = _.clone(sample.clinicalData[1]);
+        attr.clinicalAttribute = {
```
Comment (alisman) [inline src/pages/patientView/clinicalInformation/PatientViewPageStore.ts]: is this draft code?

Comment (dippindots) [reply src/pages/patientView/clinicalInformation/PatientViewPageStore.ts]: @alisman Yes, it is the dummy data I created. I will remove it later when we have the attribute in the data.

Comment (dippindots) [reply src/pages/patientView/clinicalInformation/PatientViewPageStore.ts]: I should add comments for the dummy data next time.

**Comment by dippindots** [comment]
@jjgao Yes, thanks! Just updated.

**Pull request merged**